src/loader.ts

```diff
@@ -1,10 +1,28 @@
+import path from "path";
 import type { NodeLoaderHooksAPI2 } from "ts-node";
+import { URL } from "url";
```

Comment:
Please sort these in a separate section above and use `node:` scheme for consistency.